### PR TITLE
Added Comment Input Component

### LINKED ISSUE #149: Make Comment-Input Component
Refer to the Figma design.
Make it responsive. Use correct breakpoints for media queries.
Refrain from using fixed widths and heights.
Use existing components wherever necessary.

### PR DESCRIPTION
**Pull request by priyanshisharma**
### Fixes #149  

**Description** <br/>
I've added comment input box.

**Screenshots** <br/>
Web View
Screen Shot 2020-11-06 at 1 06 59 PM

Mobile View
Screen Shot 2020-11-06 at 1 07 34 PM

On typing
Screen Shot 2020-11-06 at 1 16 15 PM

### CODE REVIEW

File: frontend/src/css/components/_comment-input.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  border: none;
+  box-shadow: none;
+  padding-left: 20px;
+}
+
+.comment-input {
+
+    &__text-container {
+      border-radius: 4px;
+      padding :  20px 10px;
```
Comment (salil-naik) [inline frontend/src/css/components/_comment-input.scss:12]: This should be 10px 20px;

File: frontend/src/css/components/_comment-input.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    &__input-comment{
+      @extend %input-field;
+
+      &:focus,&:hover{
+        @extend %input-field;
+      }
+    }
+
+    &__post{
```
Comment (salil-naik) [inline frontend/src/css/components/_comment-input.scss:35]: Make a `.button__comment` class in `_button.scss`

File: frontend/src/components/CommentInput.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    <Avatar avatarSrc={avatar_image}/>
+                </div>
+                <div className="comment-input__text-container">
+                    <input
+                        className="comment-input__input-comment"
+                        type = "text"
+                        placeholder="Write your comment here ..."
+                        onChange={this.changeHandler}
+                    />
+                    <a className="comment-input__post">
```
Comment (salil-naik) [inline frontend/src/components/CommentInput.jsx:29]: Use button tag her.

File: frontend/src/components/CommentInput.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      }
+
+    render() {
+        return(
+            <div className="flex-container">
+                <div className="comment-input__img-container">
+                    <Avatar avatarSrc={avatar_image}/>
+                </div>
+                <div className="comment-input__text-container">
+                    <input
```
Comment (salil-naik) [inline frontend/src/components/CommentInput.jsx:23]: Put input and button in `<form> </form>`

**Review by salil-naik — CHANGES_REQUESTED** [review]
Hi. Kindly do the changes. Resolve conflicts if any.

**Comment by priyanshisharma** [comment]
I've done the specified changes and fixed the conflicts, Please have a look.

**Review by salil-naik — APPROVED** [review]
Good work.

**Comment by priyanshisharma** [comment]
Thank you 😄

**Pull request merged**